src/components/Graphs/LineGraph.js

```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import {LineGraphStyle} from "./LineGraphStyle";
+
+export const LineGraph = () => {
+  return <LineGraphStyle className="line-graph"> This is a div for Line Graph </LineGraphStyle>;
```

Comment:
remove className

Reply:
Please check the updated pull request =]